### PR TITLE
Add ClassPathUtils, copied from Quarkus.

### PR DESCRIPTION
**Pull request by radcortez**

### CODE REVIEW

File: classloader/pom.xml [hunk 1]
```diff
@@ -21,6 +21,14 @@
...
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap</groupId>
+            <artifactId>shrinkwrap-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap</groupId>
+            <artifactId>shrinkwrap-impl-base</artifactId>
+        </dependency>
```
Comment (Ladicek) [inline classloader/pom.xml:31]: Aren't they both `<scope>test</scope>`? I'm pretty sure we wouldn't want all consumers to also transitively get ShrinkWrap.

Comment (radcortez) [reply classloader/pom.xml:31]: Right :) Thanks!

**Review by Ladicek — APPROVED** [review]
I vaguely recall someone saying the class shouldn't have a `*Util[s]` name, but I don't remember the specifics. LGTM anyway :-)

**Comment by radcortez** [comment]
> I vaguely recall someone saying the class shouldn't have a `*Util[s]` name, but I don't remember the specifics. LGTM anyway :-)

Yes, I also heard that before, but I'm terrible for names. Also, wanted to keep the same name as Quarkus :)

Thanks!

**Pull request merged**